test/jdk/java/foreign/StdLibTest.java

```diff
@@ -21,10 +21,21 @@
  * questions.
  */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2022, 2022 All Rights Reserved
+ * ===========================================================================
+ */
+
 /*
  * @test
  * @enablePreview
+<<<<<<< HEAD
+ * @requires ((os.arch == "amd64" | os.arch == "x86_64") & sun.arch.data.model == "64") | os.arch == "aarch64" | os.arch == "riscv64"
+ * | os.arch == "ppc64" | os.arch == "ppc64le" | os.arch == "s390x"
+=======
  * @requires jdk.foreign.linker != "UNSUPPORTED"
+>>>>>>> origin/master
```

Comment:
This file no longer needs changes for openj9.

Reply:
Fixed